website/i18n/en.json

```diff
@@ -0,0 +1,1832 @@
+        "sidebar_label": "External Displays"
+      },
+      "system-settings/key-profiles": {
+        "title": "Key Profiles",
+        "sidebar_label": "Key Profiles"
+      },
+      "system-settings/recovering-reinstalling-the-console": {
+        "title": "Recovering & Reinstalling the Console",
+        "sidebar_label": "Recovering & Reinstalling the Console"
+      },
+      "system-settings/release-notes": {
+        "title": "Release Notes",
+        "sidebar_label": "Release Notes"
+      },
+      "system-settings/the-system-menu": {
+        "title": "The System Menu",
+        "sidebar_label": "The System Menu"
+      },
+      "system-settings/upgrading-the-software": {
+        "title": "Upgrading the Software",
+        "sidebar_label": "Upgrading the Software"
+      },
+      "system-settings/user-settings": {
+        "title": "User Settings",
+        "sidebar_label": "User Settings"
+      },
+      "titan-basics": {
+        "title": "Connecting up and Titan basics",
+        "sidebar_label": "Connecting up and Titan basics"
+      },
+      "titan-basics/clearing-the-console": {
+        "title": "Clearing the Console",
+        "sidebar_label": "Clearing the Console"
+      },
+      "titan-basics/creating-reports": {
+        "title": "Creating Reports",
+        "sidebar_label": "Creating Reports"
+      },
+      "titan-basics/front-panel-buttons": {
+        "title": "Front Panel Buttons",
+        "sidebar_label": "Front Panel Buttons"
+      },
+      "titan-basics/loading-and-saving-shows": {
+        "title": "Loading and saving shows",
+        "sidebar_label": "Loading and saving shows"
+      },
+      "titan-basics/multi-user-operation": {
+        "title": "Multi-User Operation",
+        "sidebar_label": "Multi-User Operation"
+      },
+      "titan-basics/other-parts-of-the-touch-screen": {
+        "title": "Other Parts of the Touch Screen",
+        "sidebar_label": "Other Parts of the Touch Screen"
+      },
+      "titan-basics/show-library": {
+        "title": "Show Library",
+        "sidebar_label": "Show Library"
+      },
+      "titan-basics/titan-simulator": {
+        "title": "Titan Simulator",
+        "sidebar_label": "Titan Simulator"
+      },
+      "titan-basics/workspace-windows": {
+        "title": "Workspace Windows",
+        "sidebar_label": "Workspace Windows"
+      },
+      "titan-commands": {
+        "title": "Titan Commands Quick Reference",
+        "sidebar_label": "Titan Commands Quick Reference"
+      },
+      "titan-net": {
+        "title": "Titan Net Processor Operation",
+        "sidebar_label": "Titan Net Processor Operation"
+      },
+      "titan-net/tnp-console-mode": {
+        "title": "TNP Console Mode",
+        "sidebar_label": "TNP Console Mode"
+      },
+      "titan-net/tnp-slave-mode": {
+        "title": "TNP Slave Mode",
+        "sidebar_label": "TNP Slave Mode"
+      },
+      "version-12.0/version-12.0-about-the-consoles": {
```

Comment:
Should probably correct these in the `en.json` to the correct format so that it's just a copy paste job for new languages

Reply:
This also throws a load of warnings when building the search index